intel/k8s.py

```diff
@@ -213,6 +223,15 @@ def get_namespaces(config):
     return k8s_api.list_namespace().to_dict()
 
 
+# Get kubelet version from node
+def get_kubelet_version(config):
+    k8s_api = version_api_client_from_config(config)
+    dict = k8s_api.get_code()
```

Comment:
`dict` is a reserved keyword in Python, overriding it results in unexpected behavior.